doc/api/timers.md

```diff
@@ -2,89 +2,142 @@
 
-If `callback` is not a function `setInterval()` will throw immediately.
+### setTimeout(callback, delay[, arg][, ...])
 
-## setTimeout(callback, delay[, arg][, ...])
+* `callback` {Function} The function to call when the timer elapses.
+* `delay` {number} The number of milliseconds to wait before calling the
+  `callback`.
+* `[, arg][, ...]` Optional arguments to pass when the `callback` is called.
 
 Schedules execution of a one-time `callback` after `delay` milliseconds.
-Returns a `timeoutObject` for possible use with [`clearTimeout`][]. Additional
-optional arguments may be passed to the callback.
+Returns a `Timeout` object for possible use with [`clearTimeout`][].
 
-The callback will likely not be invoked in precisely `delay` milliseconds.
+The `callback` will likely not be invoked in precisely `delay` milliseconds.
 Node.js makes no guarantees about the exact timing of when callbacks will fire,
 nor of their ordering. The callback will be called as close as possible to the
 time specified.
 
-To follow browser behavior, when using delays larger than 2147483647
-milliseconds (approximately 25 days) or less than 1, the timeout is executed
-immediately, as if the `delay` was set to 1.
+When `delay` is larger than `2147483647` or less than `1`, the `delay` will be
+set to `1`.
+
+If `callback` is not a function, a [`TypeError`][] will be thrown.
+
+## Cancelling Timers
+
+The [`setImmediate`][], [`setInterval`][], and [`setTimeout`][] methods each
+return objects that represent the scheduled timers. These can be used to cancel
+the timer and prevent it from triggering.
+
+### clearImmediate(immediateObject)
+
+* `immediateObject` {Immediate} An immediate timer object as returned by
+  [`setImmediate`][].
+
+Cancels an `Immediate` object created by `setImmediate()`.
+
+### clearInterval(timeoutObject)
+
+* `timeoutObject` {Timeout} An interval timer object as returned by
+  [`setInterval`][].
 
-If `callback` is not a function `setTimeout()` will throw immediately.
+Cancels a `Timeout` object created by `setInterval()`.
 
-## unref()
+### clearTimeout(timeoutObject)
 
-The opaque value returned by [`setTimeout`][] and [`setInterval`][] also has the
-method `timer.unref()` which allows the creation of a timer that is active but
-if it is the only item left in the event loop, it won't keep the program
-running. If the timer is already `unref`d calling `unref` again will have no
-effect.
+* `timeoutObject` {Timeout} A timeout timer object as returned by
+  [`setTimeout`][].
 
-In the case of [`setTimeout`][], `unref` creates a separate timer that will
-wakeup the event loop, creating too many of these may adversely effect event
-loop performance -- use wisely.
+Cancels a `Timeout` object created by `setTimeout()`.
 
-Returns the timer.
 
+[the Node.js Event Loop]: https://github.com/nodejs/node/blob/master/doc/topics/the-event-loop-timers-and-nexttick.md
+[`TypeError`]: errors.html
```

Comment:
Should be https://nodejs.org/dist/latest-v6.x/docs/api/errors.html#errors_class_typeerror I think?